srcpkgs/dbus/template

```diff
@@ -15,7 +15,7 @@ maintainer="Juan RP <xtraeme@voidlinux.eu>"
 license="GPL-2.0-or-later"
 homepage="https://dbus.freedesktop.org/"
```

Comment:
please add the following line verbatim directly below homepage=  and directly above distfiles=

changelog="https://raw.githubusercontent.com/freedesktop/dbus/dbus-1.12/NEWS"

Reply:
changelog added between homepage and distfiles